### DISCUSSION TITLE
Difference between otel/opentelemetry-collector-contrib  vs  otel/opentelemetry-collector

### DISCUSSION THREAD

**Discussion by rabollin** [open-telemetry/opentelemetry-collector-contrib#6971] [Q&A]
We are using OTEL libraries for generating Traces, logs & metrics. There are two sets of docker images - one under otel/opentelemetry-collector-contrib  and the other is-   otel/opentelemetry-collector.   What is the difference between these 2, what is recommended for what scenarios?

Is this component GAed already? if not what is the timeline for GA.

**Comment by jpkrohling** [comment]
The core distribution contains only a handful of components, such as the OTLP receiver/exporter, batch processor, and a few receivers and exporters related to open source projects (Jaeger, Zipkin, Prometheus, ...):

https://github.com/open-telemetry/opentelemetry-collector-releases/blob/main/distributions/otelcol/manifest.yaml

The contrib is everything under the sun:
https://github.com/open-telemetry/opentelemetry-collector-releases/blob/main/distributions/otelcol-contrib/manifest.yaml
